docker/1.14/Dockerfile.eia

```diff
@@ -0,0 +1,91 @@
+FROM ubuntu:16.04
+LABEL com.amazonaws.sagemaker.capabilities.accept-bind-to-port=true
+
+ARG TFS_SHORT_VERSION=1.14
+ARG TFS_VERSION=1-14-0
```

Comment:
rename this to TFS_EI_VERSION since if might conflict with ARGS already being used in the scripts directory

Reply:
Renamed to TFS_EI_VERSION.